core/baseline-core-v1.0-psd01.md

```diff
@@ -191,8 +191,273 @@ The ability of a Party operating Workflows on a baseline-compliant implementatio
+* **DLT Layer**: This layer manages, properly sequences and deterministiaclly processes in a privacy-preserving, cryptographically verifiable manner all transactions from the Processing Layer as well as either deterministcally or probabilistically finalizes on the DLT all DLT state transitions based on said transactions.
+
+Below we list and define the components of each layer. The detailed requirements for each component will be discussed in later sections of this document.
+
+* **BPI Abstraction layer** is comprised of
+    * **API Gateway**: An API gateway which exposes all required functionality to the counterparties to an agreement and enforces all necessary authentication and authorization of API calls as well as properly directs the API calls within the Baseline Protocol Stack
+    * **Application**: The application logic which manages the pre-processing and routing of all API requests, as well as the enforcement of authentication and authoorization protocols and rules.
+* **Middleware Layer** is comprised of:
+    * **Workflows**: A Business Process Management engine that allows for the definition, management and instantiation of workflows and worksteps and associated buiness rules and data based on (commercial) agreements between counterparties
+    * **Identity/Accounts/Workgroups**: A capability that allows for the identification and management of counterparties and their delegates as well as members of workflows and worksteps organizaed in workgroups which are derived from the counterparties to an agreement. 
+    * **Messaging**: A mesaging capability that allows the exchange of secure and privacy-preserving messages between counterparties to an agreement to communicate and coordinate agreement on proposed (commercial) state changes. 
+* **Processing Layer** is comprised of
+    * **Transaction Pool**: one or more transaction pools which hold, properly sequence, preprocess and batch for processing by the Virtual State Machine all requested state change transactions of a BPI.
+    * **Virtual State Machine**: one or more Virtual State Machines which deterministiaclly processes and finalizes in a privacy-preserving, cryptographically verifiable manner all state change request transactions.
+    * **Storage**: A storage system for the cryptographically linked current and historical state of all (commercial) agreements in a BPI.
+* **DLT Abstraction Layer** is comprised of
+    * **API Gateway**: An API gateway which enables accessing all required BPI functions implemented on one or more DLTs, and properly directs the requests within the DLT Abstraction layer to the proper DLT API application logic
+    * **Application**: The DLT API application logic which manages the pre-processing, as well as the proper usage of the underlying DLT and BPI authentication and authorization protocols and rules. 
```

Comment:
do we need to say 'and rules' ? (isn't that what a protocol is anyway?)

Reply:
better be explicit.